PORTAL/jobs/_utils.py

```diff
@@ -3067,6 +3067,14 @@ def as_metadata(self):
             metadata['dnsname'] = self.dnsname
         return metadata
 
+    def __lt__(self, other):
```

Comment:
Looks like *someone* started sorting something... 😄

Reply:
It's because now we have a combination of files that have "full" `HistInfo` objects (the ones created from the new input) and "str" hosts (just `fc_linux`) which are the entries in the index table that gets read in.  Prior to this, we weren't actually generating the full metadata for the new files, so everything had strings.  The [sorting happens here](https://github.com/faster-cpython/tools/blob/main/PORTAL/jobs/_pyperformance.py#L2912).